CMakeLists.txt

```diff
@@ -156,10 +156,6 @@ install(
   DESTINATION include
 )
 
-install(FILES package.xml
-  DESTINATION share/${PROJECT_NAME}
```

Comment:
Is `share` a global directory, as in `/share`?

Reply:
`share` is a relative path here, from cmake doc:
`If a relative path is given it is interpreted relative to the value of the CMAKE_INSTALL_PREFIX variable.`
And,
`[CMAKE_INSTALL_PREFIX] defaults to /usr/local on UNIX and c:/Program Files/${PROJECT_NAME} on Windows.`

Reply:
cool thanks